lib/floe/workflow/item_processor.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Floe
+  class Workflow
+    class ItemProcessor < Floe::WorkflowBase
```

Comment:
Very cool 👍... Was wondering how you were going to implement it, since it's like a sub workflow